### PR TITLE
Updated quote command, Deps, Docs. Added zlib & zucc

### PR DESCRIPTION
**Pull request by ThomasErhel**

### CODE REVIEW

File: commands/quote.js [hunk 1]
```diff
@@ -3,20 +3,24 @@ const fetch = require('node-fetch');
 
 exports.run = async (message) => {
 const { quote, author } = await fetch('http://quotes.stormconsultancy.co.uk/random.json').then(response => response.json())
-const { file } = await fetch('https://aws.random.cat/meow').then(response => response.json())
+const { contents } = await fetch('https://quotes.rest/qod.json').then(response => response.json())
+const { data } = await fetch('https://api.giphy.com/v1/gifs/random?api_key=20SloZAgXiC07Zqu9pBwIllwoVJ9v1xn').then(response => response.json())
 const { url } = await fetch('https://api.nasa.gov/planetary/apod?api_key=DEMO_KEY').then(response => response.json())
```
Comment (crock) [inline commands/quote.js]: Can you please move the environment variable to the `.env` file? We already have the dotenv package pulled in, so once you add `GIPHY_API_KEY=` to the .env file, you can access the value in your code with `process.env.GIPHY_API_KEY`. Thanks!

File: package.json [hunk 2]
```diff
@@ -14,12 +14,14 @@
...
     "dotenv": "^8.1.0",
-    "firebase": "^7.1.0",
+    "firebase": "^7.2.0",
     "handlebars": "^4.4.3",
     "module-alias": "^2.2.2",
     "node-fetch": "2.6.0",
     "request": "^2.88.0",
-    "request-promise-native": "^1.0.7"
+    "request-promise-native": "^1.0.7",
+    "zlib-sync": "^0.1.5",
```
Comment (crock) [inline package.json]: What is `zlib-sync` and `zucc` used for because you aren't using them?

Comment (ThomasErhel) [reply package.json]: I've seen them after npm install, discordjs recommend these optionals libs but in fact i don't need them the code work already fine. i will remove these Deps

File: commands/quote.js (hunk 1 above)
Comment (ThomasErhel) [reply commands/quote.js]: Nice tips. Feel free to review my new commit when you can. Thanks!

File: package.json (hunk 2 above)
Comment (crock) [reply package.json]: Thanks

**Pull request merged**